.travis.yml

```diff
@@ -17,6 +17,12 @@ before_install:
 install:
   - pip install -r requirements-dev.txt
 
+before_script:
+  - wget http://dynamodb-local.s3-website-us-west-2.amazonaws.com/dynamodb_local_latest.tar.gz -O /tmp/dynamodb_local_latest.tar.gz
+  - tar -xzf /tmp/dynamodb_local_latest.tar.gz -C /tmp
+  - java -Djava.library.path=/tmp/DynamoDBLocal_lib -jar /tmp/DynamoDBLocal.jar -inMemory -port 8000 &
+  - sleep 2
```

Comment:
This is obviously flaky btw, but I felt it was ok. We could increase the number a bit, but I'm not too worried about having something here that's fundamentally flaky.